### ISSUE TITLE
What happens when you run out of fuel?

### ISSUE THREAD

**Issue by 7yl4r** [7yl4r/the-oregon-trajectory#55]

**Comment by terpin** [comment]
Call for help function, maybe? You lose a crew member each time you run out
of fuel though, just to keep things interesting (after all, if the game is
too impossible to win kids won't want to play it)

**Comment by JesusVidal** [comment]
That's true, but the idea of losing a member of the crew sounds a little dark, what about deciding to get in orbit of an object and searching for resources there, I mean maybe land on a passing asteroid and if you have luck, you ll find the right components for making an emergency fuel, just an idea

**Comment by 7yl4r** [comment]
I think dark is okay. The Oregon Trail was pretty dark too. As for getting fuel from asteroids, that's exactly why I am so excited to get random events in the game!

**Comment by terpin** [comment]
Haha truth. I just know we all lost a lot of the wagon party in the
original oregon trail (which part of the fun!) so i don't think it would
freak anyone out too bad.

**Comment by JesusVidal** [comment]
Well that's actually true haha, I ll think of which other random events should we add.
